<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Set_question</class>
 <widget class="QDialog" name="Set_question">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1535</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: #964F4CFF;
	color: #000000;
	border-color: #000000;

}


/*-----QLabel-----*/
QLabel
{
	background-color: #00111d;
	color: #717e87;
	border-color: #000000;

}


/*-----QPushButton-----*/
QPushButton
{
	background-color: #3de6af;
	color: #000000;
	border-style: solid;
	border-color: #000000;
	font-weight : bold;
	padding: 13px;

}


QPushButton::hover
{
	background-color: #3dffaf;
	color: #000000;
	border-style: solid;
	border-color: #000000;
	font-weight : bold;
	padding: 13px;

}


QPushButton::pressed
{
	background-color: #3dc8af;
	color: #000000;
	border-style: solid;
	border-color: #000000;
	font-weight : bold;
	padding: 13px;

}


/*-----QToolButton-----*/
QToolButton
{
	background-color: #00111d;
	color: #000000;
	border-style: solid;
	border-color: #000000;
	padding: 5px;

}


QToolButton::hover
{
	background-color: #00111d;
	color: #000000;
	border-style: solid;
	border-width: 1px;
	border-color: #01dcff;

}


QToolButton::pressed
{
	background-color: #01dcff;
	color: #000000;
	border-style: solid;
	border-width: 1px;
	border-color: #01dcff;

}


/*-----QLineEdit-----*/
QLineEdit
{
	background-color: #082336;
	color: #3e93ff;
	border-style: solid;
	border-width: 2px;
	border-color: white;
	padding: 10px;

}


QLineEdit::hover
{
	background-color: #082336;
	color: #3e93ca;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}


QLineEdit::focus
{
	
	color: #3e93ca;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}


/*-----QCheckBox-----*/
QCheckBox
{
	background-color: transparent;
	color: #717e87;
	border-color: #000000;

}


QCheckBox::indicator
{
    color: #b1b1b1;
    background-color: #00111d;
    border: 1px solid #2aaaa8;
    width: 12px;
    height: 12px;

}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>701</width>
     <height>721</height>
    </rect>
   </property>
   <property name="title">
    <string>Questions</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>571</width>
      <height>621</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="Question_2">
       <item>
        <widget class="QLabel" name="Question">
         <property name="styleSheet">
          <string notr="true">background-color:#964F4CFF;
font: 14pt &quot;Calibri&quot;;
color:black;</string>
         </property>
         <property name="text">
          <string>Qestion            :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Question_3">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Calibri&quot;;
color:white;
border-radius:13px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="Option_1">
         <property name="styleSheet">
          <string notr="true">background-color:#964F4CFF;
font: 14pt &quot;Calibri&quot;;
color:black;</string>
         </property>
         <property name="text">
          <string>Option  1         :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Option_5">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Calibri&quot;;
color:white;
border-radius:13px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="Option_2">
         <property name="styleSheet">
          <string notr="true">background-color:#964F4CFF;
font: 14pt &quot;Calibri&quot;;
color:black;</string>
         </property>
         <property name="text">
          <string>Option  2         :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Option_6">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Calibri&quot;;
color:white;
border-radius:13px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="Option_3">
         <property name="styleSheet">
          <string notr="true">background-color:#964F4CFF;
font: 14pt &quot;Calibri&quot;;
color:black;</string>
         </property>
         <property name="text">
          <string>Option  3         :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Option_7">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Calibri&quot;;
color:white;
border-radius:13px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="Option_4">
         <property name="styleSheet">
          <string notr="true">background-color:#964F4CFF;
font: 14pt &quot;Calibri&quot;;
color:black;</string>
         </property>
         <property name="text">
          <string>Option  4         :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Option_8">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Calibri&quot;;
color:white;
border-radius:13px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="Answer">
         <property name="styleSheet">
          <string notr="true">background-color:#964F4CFF;
font: 14pt &quot;Calibri&quot;;
color:black;</string>
         </property>
         <property name="text">
          <string>Answer            :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Option_9">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Calibri&quot;;
color:white;
border-radius:13px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Calibri&quot;;
border-radius:13px;</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
